docs/HowToGuides/GettingStarted.md

```diff
@@ -137,6 +137,7 @@ Double-check that running `pwd` prints a path ending with `swift`.
      ```sh
      brew install cmake ninja sccache
      ```
+     ⚠️ Since version 0.2.14, `sccache` no longer caches compile commands issued by `build-script` because of [this](https://github.com/mozilla/sccache/pull/898) change and since `build-script` adds the `-arch x86_64` argument twice.
```

Comment:
I just realized, this is in the macOS section, but this problem almost certainly affects Linux too. It may make more sense to have this warning be displayed immediately at the beginning of Installing dependencies section. Say something like:

```
⚠️ Since version 0.2.14, `sccache` no longer caches compile commands issued by `build-script` because of [sccache PR 898](https://github.com/mozilla/sccache/pull/898), since `build-script` adds the `-arch x86_64` argument twice. The instructions below may install `sccache` 0.2.14 or newer. You may want to instead download and install an older release from their [Releases page](https://github.com/mozilla/sccache/releases) until this issue is resolved.
```

Reply:
Sounds good to me.